nucypher/characters/lawful.py

```diff
@@ -366,13 +366,9 @@ def grant(self,
         self.log.debug(f"Enacting {policy} ... ")
         # TODO: Make it optional to publish to blockchain?  Or is this presumptive based on the `Policy` type?
         enacted_policy = policy.enact(network_middleware=self.network_middleware,
-                                      handpicked_ursulas=handpicked_ursulas,
-                                      publish_treasure_map=publish_treasure_map)
```

Comment:
Can't comment on the code above, but `publish_treasure_map` can be removed from the list of parameters for this method.

Reply:
✔️